README.md

```diff
@@ -1,10 +1,57 @@
-# default-SEM-model
+# model_seg_rat_axon-myelin_sem
+---
+
+## Model overview
+![image of segmentation obtained from this model](sem_model_preview.png)
+
 AxonDeepSeg default SEM model and testing image. This model works at a resolution of 0.1 micrometer per pixel and was trained on rat spinal cord data collected via a Scanning Electron Microscope (SEM).
 
+## Dependencies
+- [ivadomed](https://ivadomed.org/) commit: e6554281d07b5afef9e68ce8b02e86b02bd68363
+- [axondeepseg](https://axondeepseg.readthedocs.io/en/latest/) commit: 805868e39eddf620c9f3d60d313cadffb1121bfb
+
+## Segment (ADS)
+To segment an image using this model, use the following command in an `axondeepseg` virtual environment:
+```
+axondeepseg -t SEM -i <IMG_PATH> -s <PIXEL_SIZE>
+```
+The `-m` option can be omitted in this case because this is a default built-in model.
+
+## Train and test (ivadomed)
+
+### Clone this repository
+This model was trained with `ivadomed`, which is included in the `axondeepseg` venv. Otherwise, make sure you have it installed separately. You will need the *model_seg_rat_axon-myelin_sem.json* configuration file located in this repo.
```

Comment:
I have a question about this.
The config file in the model folder correspond to a specific commit in ivadomed (see above on line 10) and a specific dataset commit as well. So I'm not sure if it will necessarily always work on the latest ivadomed version installed in ADS (because microscopy integration in ivadomed and model training were done in-parallel and ivadomed has changed since the training).

I think that for re-training the model, we should advise to install ivadomed separately according to ivadomed documentation and checkout the specific commit.

Reply:
Yes, I agree that this would make the instructions more sustainable if the ivadomed version was to change in ADS. I added instructions to install ivadomed separately.